src/vs/platform/positronActionBar/browser/components/actionBarButton.css

```diff
@@ -36,7 +36,7 @@
 
 .action-bar-button.fade-in {
 	opacity: 0;
```

Comment:
No need for this opacity of zero since it's built into the animation. Doesn't hurt though.

Reply:
I wanted to make as few unrelated changes as possible so going to skip on these for now if that's okay.